intrpvar.h

```diff
@@ -1067,7 +1067,8 @@ PERLVAR(I, wcrtomb_ps, mbstate_t)
 /* Enough space for the reserved byte, 1 for a potential leading 0, then enough
  * for the longest representable integer plus an extra, the 3 flag characters,
  * and NUL */
-PERLVARA(I, mem_log, 1 + 1 + TYPE_DIGITS(UV) + 1 + 3 + 1, char)
+#define PERL_MEM_LOG_ARYLEN (1 + 1 + TYPE_DIGITS(UV) + 1 + 3 + 1)
```

Comment:
Is `PERL_MEM_LOG_ARYLEN` going to be used anyplace other than in the line immediately following its define?  If not, what is the rationale for this change?

Reply:
Yeah, the benefit over `sizeof(PL_mem_log) isn't quite clear to me either

Reply:
@leont - i just wanted to get the complex expression out of the PERLVARA() declaration. I dont care what the complex expression is really, and I dont think changing it is in scope of this patch.
@jkeenan - no, but that doesn't mean we shouldnt make it available.

Ultimately this is just a prep patch that makes it easier to do futher cleanups on this file. It shouldn't matter nor hurt anything to do so.

Reply:
So @Leont and @jkeenan anything further on this? Can i resolve this?

Reply:
If @Leont is satisfied, then I have no further question.

Reply:
I'd vote -0, so go ahead.